libut/Makefile

```diff
@@ -1,33 +1,41 @@
-OBJS=libut.a
-all: $(OBJS) 
+SHELL=/usr/sh
+VERSION=2.0.1
+MAJOR=2
```

Comment:
Yes, make them magic numbers. Ideally, when it's time to bump the version, I want to be able to `git grep 2.0.1` and find exactly all the places that need adjustment; and when it's *not* time to bump the version, I want the code to be simple and straightforward, with as few named entities and as little cognitive overhead as possible.